packages/ui/src/types/authenticator/form.ts

```diff
@@ -64,6 +64,16 @@ export interface FormFieldOptions {
   type?: string;
   /** Desired autocomplete HTML attribute */
   autocomplete?: string;
+  /** Desired HTML pattern type */
+  pattern?: RegExp;
```

Comment:
Looks like you may want to pick this from Input type: https://github.com/aws-amplify/amplify-ui/actions/runs/5955430849/job/16154297954?pr=4376